### PR TITLE
Add missing flag to disable the watcher

### PR DESCRIPTION
**Pull request by chrisfinazzo**

### CODE REVIEW

**Comment by envygeeks** [comment]
I personally like the way that CLI's (well some do some dont) where it's `--[no-]watch` but :shipit:

**Comment by chrisfinazzo** [comment]
To each their own, I guess...

See #3607.

File: site/_docs/configuration.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ class="flag">flags</code> (specified on the command-line) that control them.
         <p class="description">Enable auto-regeneration of the site when files are modified.</p>
       </td>
       <td class="align-center">
-        <p><code class="flag">-w, --watch</code></p>
+        <p><code class="flag">-w, --watch, --no-watch</code></p>
```
Comment (parkr) [inline site/_docs/configuration.md]: I agree with @envygeeks. Would love to see this be `--[no-]watch` or similar.

Comment (chrisfinazzo) [inline site/_docs/configuration.md]: Sure, will update.

Out of curiosity, is there a short equivalent of the `--no-watch` flag? Seems odd to only have `-w`.

Comment (parkr) [inline site/_docs/configuration.md]: > Out of curiosity, is there a short equivalent of the --no-watch flag? Seems odd to only have -w.

Nope.

**Pull request merged**